src/translations/en.json

```diff
@@ -520,6 +520,11 @@
           "hint_enable": "Missing config options? Enable advanced mode on",
           "link_profile_page": "your profile page"
         },
+        "integration_panel_move": {
+          "missing_zha": "Missing the ZHA config panel? It was moved to the ZHA entry on the",
+          "missing_zwave": "Missing the Z-Wave config panel? It was moved to the Z-Wave entry on the",
+          "link_integration_page": "integrations page"
```

Comment:
You can't concatenate strings because that's not the same in every language. (Yes I know messed up on the other one too…)